src/lib/migration-steps/action-creators.ts

```diff
@@ -330,6 +343,23 @@ const actionCreators = {
         fieldGroupId
       }
     }),
+    changeFieldGroupId: (contentTypeId: string, contentTypeInstanceId: string, fieldGroupId: string, newFieldGroupId: string, callsite): Intents.EditorLayoutChangeFieldGroupId => new Intents.EditorLayoutChangeFieldGroupId({
+      type: 'contentType/changeEditorLayoutFieldGroupId',
+      meta: {
+        contentTypeInstanceId: `contentType/${contentTypeId}/${contentTypeInstanceId}`,
+        callsite: {
+          file: callsite.getFileName(),
+          line: callsite.getLineNumber()
+        }
+      },
+      payload: {
+        contentTypeId,
+        fieldGroupId,
+        props: {
+          newFieldGroupId
```

Comment:
What's the reason to put this into a props object?

Reply:
Just to not pollute the type `RawStepPayload`, similarly to how it's used in field renaming

Reply:
Renaming a field seems to be the outlier in this file and usually all the info is put into the payload (as long as the keys are fixed).

Reply:
updated!